serpent/oecd-fullcore

```diff
@@ -17,7 +17,6 @@
 * OPyC radius   422.5e-6 m    *
 * Matrix Graphite             *
```

Comment:
Did you also want to reflect the unit conversions here?

Reply:
yes, I just fixed all the unit conversions. They were all wrong ...